src/ddmd/astbase.d

```diff
@@ -5560,24 +5560,11 @@ struct ASTBase
         }
     }
 
-    extern (C++) final class AndAndExp : BinExp
+    extern (C++) final class LogLogExp : BinExp
```

Comment:
Could use a comment explaining that `LogLogExp` (or whatever name is ultimately decided on) is for `&&` _and_ `||` expressions so contributors aren't left guessing.